docs/api/batch.rst

```diff
@@ -99,6 +99,13 @@ The response body is a list of all responses:
       ]
     ]
 
+Error codes
+:::::::::::
+
+* 200 OK: The request have been processed
```

Comment:
I don't get why 200 ok is under "error codes". Should we rename the section to "HTTP status codes" ?